azure-pipelines/compliance.yml

```diff
@@ -9,32 +13,70 @@ pr:
       - dev/*
 
 pool:
-  vmImage: windows-latest
-
-steps:
-- task: PoliCheck@2
-  inputs:
-    targetType: 'F'
-    targetArgument: '$(Build.SourcesDirectory)'
-
-- task: ComponentGovernanceComponentDetection@0
-  displayName: 'Component Detection'
-
-- task: AntiMalware@4
-  inputs:
-    InputType: 'Basic'
-    ScanType: 'CustomScan'
-    FileDirPath: '$(Build.StagingDirectory)'
-    TreatSignatureUpdateFailureAs: 'Warning'
-    SignatureFreshness: 'UpToDate'
-    TreatStaleSignatureAs: 'Error'
-
-- task: CredScan@3
-  inputs:
-    scanFolder: '$(Build.SourcesDirectory)'
+  name: VSEngSS-MicroBuild2022-1ES
 
-- task: TSAUpload@2
-  displayName: 'TSA upload'
-  inputs:
-    GdnPublishTsaOnboard: True
-    GdnPublishTsaConfigFile: '$(Build.SourcesDirectory)/azure-pipelines/TSAOptions.json'
+jobs:
+- job: 'UnrealEngine_VisualStudioTools_Compliance'
+  timeoutInMinutes: 1440
+  steps:
+    - task: CmdLine@2
+      displayName: 'Clone Unreal Engine Repository'
+      inputs:
+        script: 'git clone "https://$(token)@github.com/EpicGames/UnrealEngine" --single-branch --branch $(ue_branch) --depth 1 ue'

```

Comment:
Please capture in wiki how the token is obtained (so we now how to refresh it in future).